src/box/txn.c

```diff
@@ -66,7 +66,7 @@ const char *txn_isolation_level_aliases[txn_isolation_level_MAX] = {
 };
 
 /* Txn cache. */
-static struct stailq txn_cache = {NULL, &txn_cache.first};
+static STAILQ_HEAD(txn_cache);
```

Comment:
Nit: Let's call it simply `STAILQ`? There's no such thing as `stailq_head` and we usually name definition macros after the struct they define.

Reply:
Done.